### ISSUE TITLE
Archive repos with software heritage

### ISSUE THREAD

**Issue by physikerwelt** [theoremprover-museum/theoremprover-museum.github.io#30]
I have set up automated archival of all repos of an organisation by following https://archive.softwareheritage.org/api/1/origin/save/webhook/github/doc/. Maybe this can be done for the theoremprover-museum organisation as well?

**Comment by kohlhase** [comment]
If I remember correctly, they are doing that from their end. They were quite impressed with the museum in the very beginning.

**Comment by physikerwelt** [comment]
> If I remember correctly, they are doing that from their end. They were quite impressed with the museum in the very beginning.

Yes, they will eventually archive essentially all of GitHub, but it might take some time. With the hook, archival is done automatically on each commit.
